### ISSUE TITLE
Do not actually download JARs if only dependency information is queried

### ISSUE THREAD

**Issue by sschuberth** [jrudolph/sbt-dependency-graph#130]
It looks to me as if `sbt dependencyTree` is actually downloading the JARs, see e.g. lines like

    [info] downloading https://repo1.maven.org/maven2/commons-logging/commons-logging/1.2/commons-logging-1.2.jar ...

whereas just querying the dependencies would only require to download the POMs, not the JARs. I believe by default only the absolutely necessary files should be downloaded, and only as an option, if at all, also downloads the JARs.

The use case is that we are only interested in the dependencies and their respective licenses, but not in building the code. As such we don't need the JARs.

**Comment by jrudolph** [comment]
Yeah, good point, not sure how easy it is to prevent ivy from doing that, though.

**Comment by jrudolph** [comment]
I had a try at this in https://github.com/jrudolph/sbt-dependency-graph/tree/jr/fix-130-by-only-downloading-poms-for-update but it doesn't seem to work so far.
